site-admin/templates/admin/save.hbs

```diff
@@ -0,0 +1,122 @@
+        <li><a href="/companies">Companies</a></li>
+        <li><a href="/people">People</a></li>
+      </ul>
+    </div>
+
+    <div class="Panel__Timeline">
+      <form action="/meetups" method="post">
+        <input type="text" placeholder="episode" name="episode" value={{episode}} />
+        <input type="date" placeholder="date" name="date" />
+        <h5>Starting time</h5>
+        <input type="time" placeholder="starting time" name="startsAt" />
+        <h5>Ending time</h5>
+        <input type="time" placeholder="ending time" name="endsAt" />
+
+        <h4>Venue</h4>
+        <input type="text" placeholder="Organisation" name="venue[organisation]" />
+        <input type="text" placeholder="Address Line 1" name="venue[addr_line1]" />
+        <input type="text" placeholder="Address Line 2" name="venue[addr_line2]" />
+
+        <h4>Sessions</h4>
+        <div id="sessions">
+          <input type="text" placeholder="title">
+          <input type="time" placeholder="time">
+          <input type="text" placeholder="desc">
+          {{{dropdown "people"}}}
+          <button onclick="return addPeople(event);">Add People</button>
+        </div>
+        <button onclick="return addSession(event);">Add Session</button>
+
+        <h4>Sponsors</h4>
+        <div id="sponsors">
+          {{{dropdown "companies"}}}
+        </div>
+        <button onclick="return addSponsors(event);">Add Sponsors</button>
+
+        <h4>Supporters</h4>
+        <div id="supporters">
+          {{{dropdown "companies"}}}
+        </div>
+        <button onclick="return addSupporters(event);">Add Supporters</button>
+
+        <input type="submit" value="Save" />
+      </form>
+
+      <script>
+        window.Index = {
+          Session: 0,
+          People: 0,
+          Sponsor: 0,
+          Supporter: 0
+        };
+
+        function getNode(name, type, placeholder, val) {
+          var input = document.createElement('input');
+          input.name = name;
+          input.type = type;
+          input.placeholder = placeholder;
+          input.value = val ? val : null;
+          return input;
+        }
+
+        function addSession(e) {
+          e.stopPropagation();
+          e.preventDefault();
+
+          var sessionsElm = document.getElementById('sessions');
+          sessions.appendChild(getNode("sessions[" + window.Index.Session + "][title]", "hidden", "title", sessionsElm.children[0].value));
+          sessions.appendChild(getNode("sessions[" + window.Index.Session + "][time]", "hidden", "time", sessionsElm.children[1].value));
+          sessions.appendChild(getNode("sessions[" + window.Index.Session + "][desc]", "hidden", "desc", sessionsElm.children[2].value));
+
+          ++window.Index.Session;
+
+          window.Index.People = 0;
+        }
+
+        function addSponsors(e) {
+          e.stopPropagation();
+          e.preventDefault();
+
+          var sponsorsElm = document.getElementById('sponsors');
```

Comment:
you can try cache the `DOM selectors`. instead of storing in a `var`, bring all the selectors to an `object`. which helps you to manage the selector easily. eg:
```javascript
var selectors = {
  $sessions: document.getElementById('sessions'),
  $sponsors: document.getElementById('sponsors')
};
```